### PR TITLE
update VCWG maintenance process

### PR DESCRIPTION
**Pull request by kdenhartog**
This is an update to the VCWG maintenance process to make it line up with what's being done in the VCWG. It's still the same thing but this is more consistent with what's in that [WG repo](https://github.com/w3c/vc-data-model) as well

### CODE REVIEW

**Comment by kdenhartog** [comment]
tagging @brentzundel since I can't request reviews from him in this repo as well

File: VCWGmaintprocess.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
+
+1. For now, we will focus only on merging new errata PRs into the VC Data Model Repository,
+   but encourage activity related to new features.
+2. Once a PR is opened, chairs and editors make judgement call on whether
+   changes are substantive or editorial.
+   <dl>
+     <dt>Editorial</dt>
+     <dd>Mark with "editorial" tag, merge into branch "v1.1"</dd>
+     <dt>Substantive</dt>
+     <dd>Mark with "substantial" tag. Substantial changes are merged into separate branch "v2.0" which will be addressed when a new WG is formed. New Features stay around as an open PR.</dd>
```
Comment (brentzundel) [inline VCWGmaintprocess.md:15]: I think the tag we're using is `v2.0`

Comment (kdenhartog) [inline VCWGmaintprocess.md:15]:
```suggestion
     <dd>Mark with "v2.0" tag. Substantial changes are merged into separate branch "v2.0" which will be addressed when a new WG is formed. New Features stay around as an open PR.</dd>
```

Comment (kdenhartog) [reply VCWGmaintprocess.md:15]: Ahh yup looks like any substantive issue that's got the errata label on it, but not the editorial label will get picked up as a substantive change in the errata document.

Originally that's what I was thinking this text was trying to point out was the process for substantive changes during the maintenance WG, but I'm not sure we want to include these details now that we're not taking any more substantive changes and deferring that to v2 stuff.

What do you think should we drop this portion here or just change this label to v2.0 and keep the text?

Comment (brentzundel) [reply VCWGmaintprocess.md:15]: I'd just change the label to `v2.0` keep the text

Comment (kdenhartog) [inline VCWGmaintprocess.md:15]:
```suggestion
     <dd>Mark with "v2.0" tag. Substantial changes are merged into separate branch "v2.0" which will be addressed when a new WG is formed. New Features stay around as an open PR.</dd>
```

Comment (kdenhartog) [reply VCWGmaintprocess.md]: Done.

File: VCWGmaintprocess.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ This describes the process for the Verifiable Credentials Working Group maintena
      <dt>Editorial</dt>
      <dd>Mark with "editorial" tag, merge into branch "v1.1"</dd>
      <dt>Substantive</dt>
-     <dd>Mark with "substantial" tag. Substantial changes are merged into separate branch "v2.0" which will be addressed when a new WG is formed. New Features stay around as an open PR.</dd>
+     <dd>Mark with "v2.0" tag. Substantial changes are merged into separate branch "v2.0" which will be addressed when a new WG is formed. New Features stay around as an open PR.</dd>
```
Comment (TallTed) [inline VCWGmaintprocess.md:15]:
```suggestion
     <dd>Mark with "v2.0" tag. Substantial changes are merged into separate branch "v2.0" which will be addressed when a new WG is formed. New Features stay around as an open PRs.</dd>
```

**Comment by TallTed** [comment]
@kdenhartog — Please provide some explanation when closing without merging...

**Comment by kdenhartog** [comment]
I'm not actively participating within these group anymore so I'm not certain if this process is still correct and the PR's are almost 2 years old. It would be best if someone who's active could update and address this in a separate PR.

**Pull request closed without merging**